labot/protocolBuilder.py

```diff
@@ -139,6 +141,7 @@ def parse(t):
 
 def build():
     for t in types.values():
+        print("Parsing: " + t['name'])
```

Comment:
It was just to give a sense of progression because that script is long to execute and there was no way to know if it was actually doing something. But I will remove it if you don't like it.

Reply:
Great idea! Maybe use https://github.com/tqdm/tqdm

You can even put a text with the progressbar, don't update it with the message name because it won't be readable.

Also, a logging system wouldn't be bad, especially when botting.